rdf-star-cg-spec.html

```diff
@@ -829,9 +829,100 @@ <h2>SPARQL* Update</h2>
   <section>
     <h2>RDF* Semantics</h2>
 
-    <p>In this section, we provide a model-theoretic semantics for RDF*, by extending the one defined in [[[RDF11-MT]]] [[RDF11-MT]].</p>
+    <p>In this section, we provide a model-theoretic semantics for RDF*, based on the one defined in [[[RDF11-MT]]] [[RDF11-MT]]. More precisely, we define a mapping from RDF*'s abstract syntax into standard RDF's abstract syntax, and define the semantics of <a>RDF* graphs</a> in terms of the semantics of the mapped <a>RDF graphs</a>.</p>
```

Comment:
In one place embedded triples can be predicates, but in one other place only embedded triples as subjects or objects are handled.

Reply:
@pchampin , i read this as a proposal to:
```
s/such that neither s, p nor o is an embedded triple.
 /such that neither s nor o is an embedded triple.
 /
```

Reply:
Right; the original proposal was very general, allowing any kind of term in any position. I took the liberty to make it more aligned with the restrictions in the abstract syntax, but I let this one go.